Dockerfile_dev_environment

```diff
@@ -26,13 +26,14 @@ RUN set -x \
     python3-dev \
     python3-pip \
     libgeos-dev \
-    nodejs-legacy \
-    npm \
     openjdk-7-jdk \
     gettext \
     gcc \
     git \
     virtualenv \
+    curl \
+ && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+ && apt-get install -y nodejs \
```

Comment:
Are those changes really related to the parser changes? Could you explain what they are used for?

Reply:
Not mine, it's from bruno : https://github.com/c2corg/v6_ui/pull/1844/commits/e770aba3fc62c2f520ca7941bb8d74ab7c318db8

I can't say if it's normal to have it in this PR or not.

I'll take the time to learn git, and have organized commits in the future, but I appreciate the exception for this one, i spent more time trying to organize them, than coding. Quite a pain...

Reply:
I think this is a problem with rebase g, because these modifications have been made by myself to correct the Docker build which requires a more recent version of node

Reply:
Indeed, this change is already available in the master branch: https://github.com/c2corg/v6_ui/commit/7c3c0e7e93593becd8fe42676df65dc1d82520a1
It's a bit strange git does not report a conflict?!
I will try to propose a cleaned (separated) branch. I let you know.

Reply:
Thank you :)